python/ray/serve/api.py

```diff
@@ -96,12 +99,14 @@ def init(
         ray_init_kwargs (dict): Argument passed to ray.init, if there is no ray
             connection. Default to {"object_store_memory": int(1e8)} for
             performance stability reason
-        gc_window_seconds(int): How long will we keep the metric data in
-            memory. Data older than the gc_window will be deleted. The default
-            is 3600 seconds, which is 1 hour.
         queueing_policy(RoutePolicy): Define the queueing policy for selecting
             the backend for a service. (Default: RoutePolicy.Random)
         policy_kwargs: Arguments required to instantiate a queueing policy
+        metric_sink(BaseSink): The metric storage actor for all RayServe actors
+            to push to. RayServe have two options built in: InMemorySink and
+            PrometheusSink
+        metric_push_interval(float): The interval for each actors to push to
+            the metric sink. Default is 5s.
```

Comment:
```suggestion
            the metric sink. Default is 5s.
```
```suggestion
            the metric sink. Default is 2s.
```

Reply:
It's set to 2s below in `constants.py`. Is there any reason for having a constant and not just hardcoding it here?